packages/front-end/components/Features/FeatureModal.tsx

```diff
@@ -198,6 +214,17 @@ export default function FeatureModal({
         }
       />
 
+      <div className="form-group">
+        <label>Project</label>
+        <SelectField
+          value={form.watch("project")}
+          onChange={(p) => form.setValue("project", p)}
+          name="project"
+          initialOption={allowAllProjects ? "All Projects" : undefined}
```

Comment:
It might be nice to default to the current project if one is selected.

Reply:
It does (or should) - that's set in the form defaults when the form is initialized.

Reply:
Sorry, didn’t realize this during review